internal/conf/conftypes/conftypes.go

```diff
@@ -101,3 +102,33 @@ func (r *RawUnified) FromProto(in *proto.RawUnified) {
 func (r RawUnified) Equal(other RawUnified) bool {
 	return r.Site == other.Site && reflect.DeepEqual(r.ServiceConnections, other.ServiceConnections)
 }
+
+// Bedrock ModelIDs can either be just a <model_id> string as specified in https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html. Or when using provisioned capacity a string like `<model_id>/<provisioned_capacity>` where the <provisioned_capacity> refers to the arn in https://docs.aws.amazon.com/bedrock/latest/APIReference/API_CreateProvisionedModelThroughput.html#API_CreateProvisionedModelThroughput_RequestSyntax
+type BedrockModelIdParts struct {
```

Comment:
nit: Golang naming conventions would have you write this as ID (all caps).

```suggestion
type BedrockModelIDParts struct {
```

But I'll also say that something doesn't quite smell right, here. Since this data type isn't "parts" so much as just a way to describe an AWS Bedrock _model_. (And that just requires a `ProvisionedCapacityARN` in some situations.)

So what do you think about renaming this to just `BedrockModel` or `BedrockModelRef` or `BedrockModelReference`? "XXX ID Parts" just doesn't sound right, IMHO.